docs/Guide/application-commands/application-command-registry/acquiring-application-command-registry.mdx

```diff
@@ -45,8 +41,7 @@ export class PingCommand extends Command {
 }
 ```
 
-The method returns the [`ApplicationCommandRegistry`][application-command-registry] for the command class and is
-specifically for the class in which it's called.
+This means that this registery will only concern itself about the current `ping` class- it's not going to impact anything else.
```

Comment:
This entire paragraph makes no sense. Registries already are tied to one command name and will not impact others